src/react/deckgl.js

```diff
@@ -82,6 +82,9 @@ export default class DeckGL extends React.Component {
   }
 
   @autobind _onRendererInitialized({gl, canvas}) {
+    gl.enable(GL.BLEND)
```

Comment:
this linter is tyrranical...

Although, I have to admit this is the cleanest codebase I've ever worked on